### DISCUSSION TITLE
Requirements.txt file for all branch

### DISCUSSION THREAD

**Discussion by Viddesh1** [tusuii/All-Pracs#14] [TODO]
Hello 👋 ,

I think that it will be better if the requirements.txt files are added to all the branch including the main branch. Since, Branches are created as per the subjects in the semester's. Rather than installing the required packages and dependencies manually. Having a requirements.txt with all the dedicated packages will help for execution of the specific branch files.

### What do you think about this? 🤔

**Comment by tusuii** [comment]
yes, i agree with you on that point
